.github/workflows/tests.yml

```diff
@@ -1,6 +1,14 @@
-name: build
+name: tests
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '!*.*.*'
```

Comment:
Shouldn't be `'!**'`?

Reply:
It can be that too.
`'!**'` restricts all the tags
`'!*.*.*'` restricts the tags with the pattern `n.n.n`

Do you want me to update the PR with `'!**'`?

Reply:
Yes, please. I think it makes sense to restrict all tags.

Reply:
Okay